src/api/app/models/flag.rb

```diff
@@ -37,9 +37,9 @@ def validate_duplicates
 
   scope :with_types, ->(type) { where(flag: type) }
   scope :with_repositories, ->(repo_name) { where(repo: repo_name) }
```

Comment:
Yeah true, I basically introduced this before to be used in views where I don't really like to use `.where`. I'll change it back.